src/test/java/org/broadinstitute/consent/http/service/LibraryCardServiceTest.java

```diff
@@ -351,6 +353,38 @@ public void testFindLibraryCardById() {
     assertEquals(result.getId(), libraryCard.getId());
   }
 
+  @Test
+  public void testFindLibraryCardDaaById_NotFound() {
```

Comment:
One thing we've been doing is to upgrade our tests to JUnit5 using `@ExtendWith(MockitoExtension.class)` which helps keep the test classes cleaner. For example, none of the `public` modifiers are necessary. Also, with the extends annotation, there are a good number of tests that are stubbing incorrectly. That helps catch a lot of confusing test situations. Not a requirement for this PR, I think, since it is already heavy with unintended infrastructural changes, but we should address this perhaps in a separate ticket.